kubernetes-cluster-kubeone/terraform/run.sh

```diff
@@ -29,6 +29,21 @@ else
     echo -e "Terraform initialized: \033[32m✔️\033[0m" 
 fi
 
+# check if kubeone is installed
+if ! command -v kubeone &> /dev/null; then
+  echo "Installing Kubeone..."
+  curl -sfL https://get.kubeone.io | sh
+  if ! command -v kubeone &> /dev/null; then
+    echo "unable to install kubeone please go to https://docs.kubermatic.com/kubeone/v1.7/tutorials/creating-clusters/ and install it"
+    exit 1
+  fi
+fi
+
+# remove kubeone folder
+find . -type d -name "kubeone_*" -exec rm -rf {} +
```

Comment:
Do we always want to delete this file or only under certain circumstances?

Reply:
only after install kubeone, I already moved this code to just run once kubeone was installed